common/assets/sass/application.scss

```diff
@@ -15,13 +15,18 @@ $white-transparent: rgba(255, 255, 255, .25);
   }
 }
 
+.moj-header {
+  border-bottom: 5px solid govuk-colour("blue");
```

Comment:
should these (line 19 and 29) 5px not be a govuk var?

Reply:
I'm saying no - these are for a specific component that we don't need to change size at all - not related to page rhythms etc